src/Phinx/Console/Command/Create.php

```diff
@@ -274,10 +279,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         // inject the class names appropriate to this migration
         $classes = array(
-            '$useClassName'  => $this->getConfig()->getMigrationBaseClassName(false),
-            '$className'     => $className,
-            '$version'       => Util::getVersionFromFileName($fileName),
-            '$baseClassName' => $this->getConfig()->getMigrationBaseClassName(true),
+            '$defineNamespace' => null !== $namespace ? ('namespace ' . $namespace . ';') : '',
```

Comment:
I'm not sure I understand your proposals.

$namespace variable can contain string like `Foo\Bar`

$defineNamespace is a result of concatenation: `namespace Foo\Bar;`

When namespace for path is defined in configuration, then result string injected in template (see ./src/Phinx/Migration/Migration.template.php.dist, ./src/Phinx/Seed/Seed.template.php.dist).

If namespace for path not found, then namespace definition in template is absent ($defineNamespace var is empty).

Please clarify your sentence.

Do you mean to create separate template for migrations with namespace? I suppose it would be more convenient to leave a single template.

Or maybe you suggest rename $defineNamespace var to $namespaceDefinition?